<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSSettingsDialog</class>
 <widget class="QDialog" name="CSSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>191</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>191</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>431</width>
    <height>191</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chip Select Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/active64.png</normaloff>
    <normalon>:/icons/icons/active64.png</normalon>
    <disabledoff>:/icons/icons/greyed64.png</disabledoff>
    <disabledon>:/icons/icons/greyed64.png</disabledon>
    <activeoff>:/icons/icons/active64.png</activeoff>
    <activeon>:/icons/icons/active64.png</activeon>
    <selectedoff>:/icons/icons/selected64.png</selectedoff>
    <selectedon>:/icons/icons/selected64.png</selectedon>:/icons/icons/active64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxActiveCSValues">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Active Chip Select Values</string>
   </property>
   <widget class="QCheckBox" name="checkBoxActiveCS0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxActiveCS1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxActiveCS2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxActiveCS3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxActiveCS4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxActiveCS5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxActiveCS6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxActiveCS7">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxIdleCSValues">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Idle Chip Select Values</string>
   </property>
   <widget class="QCheckBox" name="checkBoxIdleCS0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxIdleCS1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxIdleCS2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxIdleCS3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxIdleCS4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxIdleCS5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxIdleCS6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxIdleCS7">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>checkBoxActiveCS0</tabstop>
  <tabstop>checkBoxActiveCS1</tabstop>
  <tabstop>checkBoxActiveCS2</tabstop>
  <tabstop>checkBoxActiveCS3</tabstop>
  <tabstop>checkBoxActiveCS4</tabstop>
  <tabstop>checkBoxActiveCS5</tabstop>
  <tabstop>checkBoxActiveCS6</tabstop>
  <tabstop>checkBoxActiveCS7</tabstop>
  <tabstop>checkBoxIdleCS0</tabstop>
  <tabstop>checkBoxIdleCS1</tabstop>
  <tabstop>checkBoxIdleCS2</tabstop>
  <tabstop>checkBoxIdleCS3</tabstop>
  <tabstop>checkBoxIdleCS4</tabstop>
  <tabstop>checkBoxIdleCS5</tabstop>
  <tabstop>checkBoxIdleCS6</tabstop>
  <tabstop>checkBoxIdleCS7</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CSSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CSSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
